pkg/operator/assets/bindata.go

```diff
@@ -1363,7 +1363,7 @@ data:
     # GCP Specific
     - name: GCPInvalidProjectID
       searchRegexStrings:
-      - "platform\.gcp\.project.* invalid project ID"
+      - "platform.gcp.project.* invalid project ID"
```

Comment:
I'm curious - is there a way to test these?

Reply:
Also, this file belongs in the previous commit

Reply:
I try to test anything I'm not sure will work in the installlogmonitor tests, however in this case I included slashes here and not in the unit test. :(

I also realized this configmap is kinda misguided right now and filed this jira this morning: https://issues.redhat.com/browse/CO-1301 If we implement that change, and hardcode these regexes in our code, we could unit test them all.